rocAL/include/loaders/circular_buffer.h

```diff
@@ -31,8 +31,8 @@ THE SOFTWARE.
 #include "commons.h"
 #include "device_manager.h"
 #include "device_manager_hip.h"
-struct decoded_image_info {
-    std::vector<std::string> _image_names;
+struct DecodedDataInfo {
+    std::vector<std::string> _data_names;
```

Comment:
please rename crop_image_info to CropImageInfo

Reply:
Renamed it